spectator-web-spring/src/main/java/com/netflix/spectator/controllers/filter/MeasurementFilter.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spectator.controllers.filter;
+
+import com.netflix.spectator.api.Measurement;
+import com.netflix.spectator.api.Meter;
+
+/**
+ * Interface for filtering measurements.
+ *
+ * This is used to reduce available metrics down to those of interest to a caller.
+ */
+public interface MeasurementFilter {
+  /**
+   * Meter is the source of the measurement.
+   * Measurement is the particular measurement being filtered.
+   *
+   * The concrete filter determines the criteria of what to keep.
+   * Returns true to keep, false to discard.
+   */
+  boolean keep(Meter meter, Measurement measurement);
```

Comment:
I could do this. While I dont actually use meter yet, I was thinking that the meter has additional attributes such as the kind that could be filtered on so if I were to add an interface, it should include that. Measurements dont know the meter they came from so I have the interface take them both..